tests/runtime/array

```diff
@@ -142,3 +142,15 @@ RUN {{BPFTRACE}} -e 'struct A { int x[4]; } uprobe:./testprogs/array_access:test
 EXPECT Result: 1
 TIMEOUT 5
 AFTER ./testprogs/array_access
+
+NAME array compare eq
+PROG struct A { int x[4]; } uprobe:./testprogs/array_access:test_arrays { $a = (struct A *) arg0; $b = (struct A *) arg0; if ($a == $b) { printf("two int arrays are equal.\n"); } exit(); }
```

Comment:
This is not correct, you're comparing pointers (`struct A *`) instead of arrays. Should be `$a->x == $b->x`.